commerce/domain/orchestrators/classes/CartCalculateSample.cls

```diff
@@ -31,12 +31,12 @@ global class CartCalculateSample extends CartExtension.CartCalculate {
         // Use BuyerActions to decide which calculators to invoke
         CartExtension.BuyerActions buyerActions = request.getBuyerActions();
         boolean isCouponAppliedInCheckout = isCouponAppliedInCheckout(buyerActions, cart);
-        boolean runPricing = buyerActions.isCheckoutStarted() || buyerActions.isCartItemChanged();
-        boolean runPromotions = buyerActions.isCheckoutStarted() || buyerActions.isCouponChanged() || buyerActions.isCartItemChanged();
-        boolean runInventory = buyerActions.isCheckoutStarted();
-        boolean runShipping = buyerActions.isDeliveryGroupChanged() || isCouponAppliedInCheckout;
-        boolean runPostShipping = buyerActions.isDeliveryGroupChanged() || buyerActions.isDeliveryMethodSelected() || isCouponAppliedInCheckout;
-        boolean runTaxes = buyerActions.isDeliveryGroupChanged() || buyerActions.isDeliveryMethodSelected() || isCouponAppliedInCheckout;
+        boolean runPricing = buyerActions.isRecalculationRequested() || buyerActions.isCheckoutStarted() || buyerActions.isCartItemChanged();
+        boolean runPromotions = buyerActions.isRecalculationRequested() || buyerActions.isCheckoutStarted() || buyerActions.isCouponChanged() || buyerActions.isCartItemChanged();
+        boolean runInventory = buyerActions.isRecalculationRequested() && CartExtension.CartStatusEnum.CHECKOUT == cart.getStatus() || buyerActions.isCheckoutStarted();
```

Comment:
The Default CCA Orchestrator only runs the Inventory Calculator when Checkout is started, should we keep the behavior the same in this sample?

Reply:
The checkout is checked using the cart status in this case.